### DISCUSSION TITLE
Bootable ISO images

### DISCUSSION THREAD

**Discussion by VitaliyYakob** [BlueSCSI/BlueSCSI-v2#71] [Help] [answered]
Hi,
Is it supported to boot from bootable images?
I have several images that work fine if burned out to disk, but they don't boot into BlueSCSI.
The universal image of Mac OS 9.2.2 from the MacOS9Lives community is booted, but I can work with it, because this Power Mac 9600 does not have native support for 9.2.2.

I named the boot image like "CD2-Name.iso"
I don't see anything in the BlueSCSIv2 log that says errors or problems.

After the OS is loaded, the specified ISO is mounted on the system and works normally.
If you specify this image in the Startup disk, the boot is also not performed.

**Comment by erichelgeson** [comment, marked as answer]
Hi there,

Bootable images can be tricky - not all images are created equal. There are many bad rips out there. Which is why I made this page https://github.com/BlueSCSI/BlueSCSI-v2/wiki/BlueSCSI-Images#known-good-macintosh-bootable-images

Note I have successfully booted the MacOS9 lives CD - but not sure if there has been any changes or updates since I created the page. https://github.com/BlueSCSI/BlueSCSI-v2/wiki/BlueSCSI-Images#mac-os-922

It really sounds like you're doing what is required. I'd start with the basics and do a "Full overwrite" format with the SD Card Formatter utility - it sometimes takes care of weird issue and is easy to try.

Could you link to and provide a md5 or hash of the file you're using? I just got a 2nd 9600 at VCF MW and it's on the bench right now so I could test.

Thanks

**Comment by VitaliyYakob** [comment]
Hmm, Indeed, I downloaded Mac OS 7.6.1 from your page and the image of Mac OS 7.6.1.toast I am bootable and works fine.
Apparently, the ISO images I received on other resources do not contain what is needed to booting via BlueSCSI.

I'll delete the old images so as not to face these oddities anymore.

Thanks!

**Reply by erichelgeson** [reply]
> Apparently, the ISO images I received on other resources do not contain what is needed to booting via BlueSCSI.

To clarify this point a bit - BlueSCSI knows nothing about the iso or file - it just serves the bits the computer requests them over SCSI. Many ISO's or toast images are ripped incorrectly. When they are ripped they are not padded correctly, or strip information from the actual bits that are on the CD (many toast images are just the HFS volume part of the cd, not the actual CD).

If you could provide the actual file you tried I can peek at it and see what, if anything, is wrong with the rip.

**Comment by VitaliyYakob** [comment]
I think I understood what happened.
I converted some images from toast to iso and apparently made a mistake.
Now I've downloaded several boot images from different resources again -- and it works!
He created a problem for himself.

Thank you for helping me figure it out.

**Reply by erichelgeson** [reply]
Glad you got it working!
